### ISSUE TITLE
Cannot flash device

### ISSUE THREAD

**Issue by Iamyahodfpv** [Qrome/QLiteOSD#6]
I downloaded the drivers and the device shows up as com3 but when I try to flash nothing happens. Also when plugged in the blue led flashes and then turns off.

**Comment by Qrome** [comment]
Make sure you are using a data cable and not just a charging cable.   Try a new cable and a different USB Port.  This is not a software issue.

**Comment by Iamyahodfpv** [comment]
I've tried different cables and I am using a data cable. Is it possible that I bricked it? Also can it be flashed through arduino IDE or does it have to be flashed with the PYflasher?

**Comment by Iamyahodfpv** [comment]
Tried uninstalling and reinstalling the Pyflasher and the serial port showed up but now when I try to flash it is stuck on connecting

**Comment by Iamyahodfpv** [comment]
![image](https://github.com/Qrome/QLiteOSD/assets/101999008/4d076062-b37a-448e-a526-5e64fb3891e0)

**Comment by Qrome** [comment]
Remove the board from the socket and just load it to the wemos d1 mini ... I have seen bridged soldering cause the board to not connect.

**Comment by Qrome** [comment]
Yes, you can compile and load the firmware using Arduino IDE.

**Comment by Iamyahodfpv** [comment]
Yep that did the trick. Now just need to figure out what's bridged

**Comment by Qrome** [comment]
Post some photos of the top and bottom of the main board.   Did you do the button for the Wifi mode?   Check that the D3 and GND are not connected.
